src/models/m_identity.erl

```diff
@@ -542,22 +543,34 @@ get_rsc(Id, Type, Context) ->
     z_db:assoc_row("select * from identity where rsc_id = $1 and type = $2", [Id, Type], Context).
 
 
-%% @doc Hash a password, using sha1 and a salt
+%% @doc Hash a password, using bcrypt
 %% @spec hash(Password) -> tuple()
 hash(Pw) ->
-    Salt = binary_to_list(z_ids:id(10)),
-    Hash = crypto:hash(sha, [Salt,Pw]),
-    {hash, Salt, Hash}.
+    {bcrypt, erlpass:hash(Pw)}.
 
 
+%% @doc Hash a password, with bcrypt
+%% @spec bcrypt_hash(Password) -> tuple
+
 %% @doc Compare if a password is the same as a hash.
 %% @spec hash_is_equal(Password, Hash) -> bool()
+hash_is_equal(Pw, {bcrypt, Hash}) ->
```

Comment:
There are more problems with the tests. For instance, it looks like `m_rsc_db_tests` is only compiled, but not run. Also simply adding embedded unit-tests doesn't do anything either. To be able to test upgrading the hash algorithm I need a db tests. 

So I will only make tests for things I can test without a working database connection.